resources/iTerm2/readme.md

```diff
@@ -0,0 +1,29 @@
+# Remedy - Terminal
```

Comment:
I think this should be `Remedy - iTerm 2`, right?

Reply:
Fixed with 06f888a!